clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -59,6 +61,14 @@ func InitFlags(p *config.PrometheusConfig) {
 	flags.StringEnvVar(&p.ManifestPath, "prometheus-manifest-path", "PROMETHEUS_MANIFEST_PATH", "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests", "Path to the prometheus manifest files.")
 }
 
+func ValidatePrometheusFlags(p *config.PrometheusConfig) *clerrors.ErrorList {
+	errList := clerrors.NewErrorList()
+	if *shouldSnapshotPrometheusDisk && p.SnapshotProject == "" {
+		errList.Append(fmt.Errorf("Requesting snapshot, but snapshot project not configured. Use --experimental-snapshot-project flag"))
```

Comment:
nit: Errorf() should start with small letter (as those are usually joined) - this is the convension;

so s/Requesting/requesting/

Reply:
Done